js/main.js

```diff
@@ -5193,23 +5211,61 @@ function getScheduleAndTracks() {
 	return {schedule: schedule, tracks: tracks};
 }
 
-function updateSchedule() {
-    var DATE_ONE = "2017-03-18";
-    var DATE_TWO = "2017-03-19";
-    var API_VERSION = "0.0.1";
-
-    var response = getScheduleAndTracks();
-    var schedule = response.schedule[API_VERSION][0];
-    var track_halls = response.schedule[API_VERSION][0]["tracks"];
-    var tracks = response.tracks[API_VERSION][0];
-    var day_1_schedule = schedule[DATE_ONE];
-    var day_2_schedule = schedule[DATE_TWO];
+var talk_count = 0;
+var DATE_ONE = "2017-03-18";
+var DATE_TWO = "2017-03-19";
+var API_VERSION = "0.0.1";
+var response = getScheduleAndTracks();
```

Comment:
This was not required, revert your last commit and just remove var from response & that's it. Sorry if I was not clear earlier.